test/phoenix_live_view/html_tokenizer_test.exs

```diff
@@ -504,7 +586,15 @@ defmodule Phoenix.LiveView.HTMLTokenizerTest do
     end
 
     test "raise on incomplete attribute expression (EOF)" do
-      assert_raise ParseError, "nofile:2:15: expected closing `}` for expression", fn ->
+      message = """
+      nofile:2:15: expected closing `}` for expression
+        |
+      1 | <div
+      2 |   class={panel
+        |                ^\
```

Comment:
Could this be wrong?

Reply:
I guess it is because we need to scape the `\`. But this doesn't work 🤔 


```elixir
        <div
          class={panel\\
        """)
```

Reply:
I mean, if I got it right, the `\` should appear in the error message, right?

Reply:
No, the escaping has no effect here. It is only to avoid the new line. It seems we are counting the position of the new line characters in the cursor position.

Reply:
ohh yeah. Fixing it 🙌🏻